tests/device_tests/bitcoin/test_signtx_amount_unit.py

```diff
@@ -73,3 +82,50 @@ def test_signtx(client: Client, amount_unit):
         hash_link="https://tbtc1.trezor.io/api/tx/65047a2b107d6301d72d4a1e49e7aea9cf06903fdc4ae74a4a9bba9bc1a414d2",
         tx_hex="010000000001019e2123e595e20d6aa2c8d227b2b98c781b8b41fda635756eca0768b8ce8067b30000000000ffffffff02409c00000000000017a9147a55d61848e77ca266e79a39bfc85c580a6426c98750c3000000000000160014cd4ee15090b177f15ed0760d85956d93b5990d5f0247304402200c734ed16a9226162a29133c14fad3565332c60346050ceb9246e73a2fc8485002203463d40cf78eb5cc9718d6617d9f251b987e96cb58525795a507acb9b91696c7012103f60fc56bf7b5326537c7e86e0a63b6cd008eeb87d39af324cee5bcc3424bf4d000000000",
     )
+
+
+@pytest.mark.parametrize("amount_unit", VECTORS)
+def test_signtx_btc(client: Client, amount_unit):
+    # input tx: 0dac366fd8a67b2a89fbb0d31086e7acded7a5bbf9ef9daa935bc873229ef5b5
+
+    inp1 = messages.TxInputType(
+        address_n=parse_path("m/44h/0h/5h/0/9"),  # 1H2CRJBrDMhkvCGZMW7T4oQwYbL8eVuh7p
+        amount=63_988,
+        prev_hash=TXHASH_0dac36,
+        prev_index=0,
+    )
+
+    out1 = messages.TxOutputType(
+        address="13Hbso8zgV5Wmqn3uA7h3QVtmPzs47wcJ7",
+        amount=50_248,
+        script_type=messages.OutputScriptType.PAYTOADDRESS,
+    )
+
+    with client:
+        client.set_expected_responses(
+            [
+                request_input(0),
+                request_output(0),
+                messages.ButtonRequest(code=messages.ButtonRequestType.ConfirmOutput),
+                messages.ButtonRequest(code=messages.ButtonRequestType.SignTx),
+                request_input(0),
+                request_meta(TXHASH_0dac36),
+                request_input(0, TXHASH_0dac36),
+                request_output(0, TXHASH_0dac36),
+                request_output(1, TXHASH_0dac36),
+                request_input(0),
+                request_output(0),
+                request_output(0),
+                request_finished(),
+            ]
+        )
+
```

Comment:
please exclude this block, that's not what we are interested in here so it's just needlessly spamming the test size

Reply:
0dede9b8d613477ea75203e19d1448945b6fd6ad